sdk/python/kfp_tekton/compiler/__init__.py

```diff
@@ -47,9 +47,13 @@ def monkey_patch():
 
 
 try:
-    # print("Applying monkey patch")
+    print("Applying KFP-Tekton compiler patch")
```

Comment:
uncommenting this debug output to get a quick reminder of the money patch is triggered. I had unintended results when calling the KFP `dsl-compile` command on our examples which had a top-level import of the `TektonCompiler` which triggered the monkey-patch